crate/kmip/src/openssl/private_key.rs

```diff
@@ -176,11 +176,23 @@ fn ec_private_key_from_scalar(
     curve: &RecommendedCurve,
 ) -> Result<PKey<Private>, KmipError> {
     let (nid, privkey_size) = match curve {
+        // P-CURVES
         RecommendedCurve::P256 => (Nid::X9_62_PRIME256V1, 32),
         RecommendedCurve::P192 => (Nid::X9_62_PRIME192V1, 24),
```

Comment:
```suggestion
        #[cfg(not(feature = "fips"))]
        RecommendedCurve::P192 => (Nid::X9_62_PRIME192V1, 24),
```

Reply:
done in 4d5a656a2044162d9356e159bc8723c274db3497